warp10/src/main/java/io/warp10/script/functions/TEST2.java

```diff
@@ -0,0 +1,42 @@
+// WILL BE REMOVED BEFORE PR MERGE
+//
+
+package io.warp10.script.functions;
+
+import io.warp10.continuum.gts.COWList;
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+
+import java.util.BitSet;
+
+public class TEST extends NamedWarpScriptFunction implements WarpScriptStackFunction {
```

Comment:
typo in name

Reply:
I git rm -f this file and others before commit, I cannot understand why it was pushed.